### PR TITLE
Add char_length and character_length functions

### PR DESCRIPTION
**Pull request by gem-neo4j**
Add docs for the new size(STRING) aliases

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by ncordon — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/functions/index.adoc [hunk 1]
```diff
@@ -94,6 +94,14 @@ These functions return a single value.
...
 | Function | Signature | Description
 
+1.1+| xref::functions/scalar.adoc#functions-char_length[`char_length()`]
+| `char_length(input :: STRING) :: INTEGER`
+| Returns the number of Unicode characters in a `STRING`.
+
+1.1+| xref::functions/scalar.adoc#functions-character_length[`character_length()`]
+| `character_length(input :: STRING) :: INTEGER`
+| Returns the number of Unicode characters in a `STRING`.
+
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/index.adoc]:
```suggestion
1.1+| xref::functions/scalar.adoc#functions-char_length[`char_length()`]
| `char_length(input :: STRING) :: INTEGER`
| Returns the number of Unicode characters in a `STRING`.
label:new[Introduced in 5.13]

1.1+| xref::functions/scalar.adoc#functions-character_length[`character_length()`]
| `character_length(input :: STRING) :: INTEGER`
| Returns the number of Unicode characters in a `STRING`.
label:new[Introduced in 5.13]
```

File: modules/ROOT/pages/functions/scalar.adoc [hunk 2]
```diff
@@ -36,6 +36,132 @@ CREATE
 ////
 
 
+[[functions-char_length]]
+== char_length()
+
+The function `char_length()` returns the number of Unicode characters in a `STRING`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/scalar.adoc]:
```suggestion
== char_length()

_This feature was introduced in Neo4j 5.13._

The function `char_length()` returns the number of Unicode characters in a `STRING`.
```

File: modules/ROOT/pages/functions/scalar.adoc [hunk 3]
```diff
@@ -36,6 +36,132 @@ CREATE
...
+|===
+
+The number of Unicode characters in the string is returned.
+
+======
+
+[[functions-character_length]]
+== character_length()
+
+The function `character_length()` returns the number of Unicode characters in a `STRING`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/scalar.adoc]:
```suggestion
== character_length()

_This feature was introduced in Neo4j 5.13._

The function `character_length()` returns the number of Unicode characters in a `STRING`.
```

**Review by JPryce-Aklundh — COMMENTED** [review]
Looks good - just some suggestions to add in the release information.

**Review by JPryce-Aklundh — APPROVED** [review]

**Pull request merged**